server/getAudio.js

```diff
@@ -0,0 +1,42 @@
+//Endpoint to handle requests to access participant audio data
+//I assume requests that get to this point are authorized because:
+//  -they pass OnlyAllowResearchers to get to this endpoint so token is valid
+//  -the id parameter is taken from data returned from dataEndpoint so checkAccess was passed
+//grabs wav file from s3 and returns audio in response body
+function audioEndpoint(pool, s3, request, response) {
+  const {id} = request.params;
+
+  return insecureStreamAudioFileFromS3({s3,id},request,response);
+}
+
+function getAudioKey(request, id) {
+  const deploymentFolder = [process.env.NODE_ENV, request.headers.host].join('_');
+  return [
+    deploymentFolder,
+    'wav-responses',
+    `${id}.wav`
+  ].join('/');
+}
+
+function insecureStreamAudioFileFromS3(params, request, response) {
```

Comment:
@keving17 Import this rather than copying and pasting?

Reply:
Oh yeah, I originally copied this function over because I couldnt figure out how to use it and ended up changing it a bunch. I had a lot of problems with this initially as seen in https://github.com/mit-teaching-systems-lab/threeflows/issues/354.

Since I ended up just using the original function, I'll import this instead. Should I reopen this branch to make this change or just do it in a patch on master?

Reply:
@keving17 sounds good!  for new changes, make a new branch for each set of changes.